src/Microsoft.VisualStudio.ProjectSystem.Managed.TestServices/Build/ProjectRootElementFactory.cs

```diff
@@ -7,8 +7,13 @@ namespace Microsoft.Build.Construction
 {
     internal static class ProjectRootElementFactory
     {
-        public static ProjectRootElement Create(string xml = "<Project/>")
+        public static ProjectRootElement Create(string xml = null)
```

Comment:
Why this change?

Reply:
When I use the IProjectAccessorFactory, its default value is null and that gets passed to the ProjectRootElementFactory. Because my tests use IProjectAccessFactory and I don't pass an argument, it's null therefore ProjectRootElementFactory needs to gracefully handle it. Technically I could pass "<Project/>", but I chose not to. I can change if need be.